rpcs3/rpcs3qt/debugger_frame.cpp

```diff
@@ -733,17 +743,21 @@ void debugger_frame::UpdateUI()
 			m_last_pc = cia;
 			DoUpdate();
 
-			if (cpu->state & s_pause_flags)
+			const bool paused = !!(cpu->state & s_pause_flags);
+
+			if (paused)
 			{
 				m_btn_run->setText(RunString);
-				m_btn_step->setEnabled(true);
-				m_btn_step_over->setEnabled(true);
 			}
 			else
 			{
 				m_btn_run->setText(PauseString);
-				m_btn_step->setEnabled(false);
-				m_btn_step_over->setEnabled(false);
+			}
+
+			if (is_using_interpreter(cpu->id_type()))
```

Comment:
So if it's not using interpreter, the step buttons stay enabled?

Reply:
They stay disabled.

Reply:
See EnableButtons.